### PR TITLE
Friendly querying functions and WFS endpoint dl/load function

### PR DESCRIPTION
**Pull request by lawinslow**

### CODE REVIEW

File: R/item_get_wfs.R [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+#' @export
+item_get_wfs = function(sb_id, ..., session){
+	
+	if(!requireNamespace("gdalUtils") || !requireNamespace("rgdal")){
+		stop('gdalUtils and rgdal packages not installed. 
+					Both are required to interact with WFS services. 
+				 	Please run: install.packages(c(\'rgdal\',\'gdalUtils\'))')
+	}
+	
+	wfs_url = item_get_wfs_url(sb_id)
```
Comment (jread-usgs) [inline R/item_get_wfs.R]: do you want a `stop()` if the item doesn't actually have a wfs?

Comment (lawinslow) [inline R/item_get_wfs.R]: It's a big buried, but there's a stop down on line 68

Comment (jread-usgs) [inline R/item_get_wfs.R]: :+1:

**Comment by jread-usgs** [comment]
a weird gotcha that I don't think applies here (but I will mention it because of how frustrating it is) is that WFS version 1.0.0 and version 1.1.0 return lat/lon in different orders. So if you do a 
```r
query <- 'http://cida.usgs.gov/gdp/geoserver/wfs?service=WFS&request=GetFeature&typeName=sample:CONUS_states&outputFormat=shape-zip&version=1.0.0'
file <- GET(query, write_disk(destination, overwrite=T), progress())
shp.path <- tempdir()
unzip(destination, exdir = shp.path)
states = readOGR(shp.path, layer='CONUS_states') %>% 
  spTransform(CRS(plot.CRS))
```
the file is correct, vs 
```r
service=WFS&request=GetFeature&typeName=sample:CONUS_states&outputFormat=shape-zip&version=1.1.0'
```
has issues with read in because of that lat/lon thingy. Only errors when your lats end up < 90° or > 90°...

File: R/item_get_wfs.R [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	
+	#drop the bounding box and footprint layers, which we don't want
+	layer_names = layer_names[!is.na(layer_names) & !layer_names %in% c('sb:boundingBox', 'sb:footprint')]
+	if(length(layer_names) > 1){
+		stop('SB Item WFS has > 1 layer. item_download_sp currently cannot handle more than one layer')
+	}
+	
+	fname = tempfile(fileext = '.shp')
+	
+	gdalUtils::ogr2ogr(wfs_url, fname, layer_names)
```
Comment (jread-usgs) [inline R/item_get_wfs.R]: does `wfs_url` specify WFS version?

Comment (lawinslow) [inline R/item_get_wfs.R]: @jread-usgs I'm pulling directly from the default URL for SB. It seems to be v1.0.0. Should I try forcing that to v1.1.0?

Comment (jread-usgs) [inline R/item_get_wfs.R]: I think it *should* be 1.0.0, but what if they change the default output..?

Comment (lawinslow) [inline R/item_get_wfs.R]: Leaving as-is for now per discussion with @jread-usgs

**Pull request merged**